docs/technical-reference/Testing Framework.md

```diff
@@ -51,18 +51,22 @@ TestCase end event
 TestCase start event
    ...
 ```
-The difficulty here is that a lot happens between 'testcase end' and 'testcase start' event. 
-At coverage capture it is a problem, because there is a risk of spilling coverage information to the next tests.
-And during execution phase, it is impossible to predict when the test will really be over. 
+The difficulty here is that a lot happens between `testcase end` and `testcase start` events. 
+At coverage capture it is a problem, because there is a risk of spilling coverage information to the next tests: Stryker
+can only capture coverage information on `testcase end` events, and only in association with the current running test.
+So, every mutants covered between `testcase end` and `testcase start` will be associated with the **next** test.
 
-Also, if a mutation ends up changing a test case name - typically by changing the result of `ToTring()`, this
-testcase will only run when running **all tests** and can no longer be executed in isolation.
+And during execution phase, it is impossible to predict when the test will really be over, so it is diffult to
+establish is the test was succesful.  
 
+Also, if a mutation ends up changing a test case name - typically by changing the result of `ToTring()`, it will change the
```

Comment:
```suggestion
Also, if a mutation ends up changing a test case name - typically by changing the result of `ToString()`, it will change the
```

Reply:
I've never thought about this. Is it an idea to skip mutating ToString methods to prevent this?

Reply:
1) I enclosed detection logic that ensure any such offending mutants will be run against all tests
2) it can only happens for theories, which raises many concerns about testing in general

So, I think we don't need it, and its not worth it :-)